src/Tasks/Microsoft.NET.Build.Tasks/RuntimeOptions.cs

```diff
@@ -11,6 +11,9 @@ internal class RuntimeOptions
     {
         public string tfm { get; set; }
 
+        [JsonProperty(DefaultValueHandling = DefaultValueHandling.Ignore)]
+        public string rollForward { get; set; }
```

Comment:
Nit: Pre-existing, but I'm not sure why some things are camelCased and other things are PascalCased. See tfm vs Framework(s). It seems that everything will end up camelCased regardless as we're passing some CamelCase thing to JSON.NET? If so, I'd prefer that we used .NET PascalCased convention throughout.

Reply:
Good point - didn't notice the magical converter.
I've fixed `tfm` -> `TFM` as well. The camel case conversion seems to convert that to `tfm`.. interesting, but it works.

Reply:
nit: I would go with Tfm. Framework design guideline for acronyms with more than two letters. E.g. System.Xml, not System.XML.